component/component.go

```diff
@@ -116,6 +114,20 @@ type Execer interface {
 	ExecFunc() interface{}
 }
 
+// LogPlatform is responsible for start the logs plugin that allows a plugin
+// to read logs for a deployment in it's own way.
```

Comment:
small nit since these are docs
```suggestion
// LogPlatform is responsible for starting the logs plugin that allows a plugin
// to read logs for a deployment in its own way.
```